cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -31,6 +31,9 @@ spec:
   containers:
   - name: kube-proxy
     image: {{pillar['kube_docker_registry']}}/kube-proxy:{{pillar['kube-proxy_docker_tag']}}
+    resources:
+      requests:
```

Comment:
Yeah none of the master components set limits, for the same reason. If kube-proxy on a node is really slow it could impact a cluster spanning service, so I think we should leave it as request for now and revisit if required.